packages/devtools_app/lib/src/config_specific/drag_and_drop/drag_and_drop.dart

```diff
@@ -66,4 +109,13 @@ abstract class DragAndDropState extends State<DragAndDrop> {
   void drop() {
     _dragging.value = false;
   }
+
+  bool coordinatesInBound(double x, double y) {
+    final RenderBox renderBox = context.findRenderObject();
+    final globalToLocalOffset = renderBox.globalToLocal(Offset(x, y));
+    return globalToLocalOffset.dx >= 0.0 &&
```

Comment:
using the constraints maxWidth doesn't seem right. 
https://api.flutter.dev/flutter/rendering/RenderBox/paintBounds.html
or semantic bounds would probably be better. The rectangle class should also have a helper to check intersection.